### PR TITLE
Campaign card fix [WAIV-1063]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/CampaignCardHeader/CampaignCardHeader.js [hunk 1]
```diff
@@ -49,22 +49,23 @@ const CampaignCardHeader = ({ intl, campaignData, isDetails }) => {
...
-            <div className="CampaignCardHeader__user-card-username">{`@${campaignData.guide.name}`}</div>
+            <div className="Username">{`@${campaignData.guide.name}`}</div>
           </Link>
-        </div>
-        <div className="CampaignCardHeader__user-card-total-paid">
-          <div>{intl.formatMessage({ id: 'paid', defaultMessage: 'Total paid' })}</div>
-          <div>{`${campaignData.guide.total_payed} STEEM`}</div>
+          <div className="Total-paid">
+            <div>{intl.formatMessage({ id: 'paid', defaultMessage: 'Total paid' })}</div>
+            <div className="Total-paid__colon">:</div>
```
Comment (milklim) [inline src/client/rewards/CampaignCardHeader/CampaignCardHeader.js]: Do not use capital letters for common css classes.

File: src/client/rewards/Campaign/Campaign.js [hunk 2]
```diff
@@ -23,9 +23,12 @@ const Campaign = ({ proposition, filterKey, history, intl }) => {
         ? `${(currentUSDPrice * proposition.max_reward).toFixed(2)} USD`
         : `${proposition.max_reward} STEEM`
       : '';
-  const goToProducts = () => history.push(`/rewards/${filterKey}/${requiredObject.id}`);
+  const goToProducts = () => {
+    history.push(`/rewards/${filterKey}/${requiredObject.id}`);
+  };
   return (
     <div className="Campaign">
+      {console.log(requiredObject)}
```
Comment (milklim) [inline src/client/rewards/Campaign/Campaign.js]: remove

File: src/client/rewards/CampaignCardHeader/CampaignCardHeader.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/CampaignCardHeader/CampaignCardHeader.js]: ++

File: src/client/rewards/Campaign/Campaign.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/Campaign/Campaign.js]: ++

**Pull request merged**